src/components/video-player/video-controls/VideoControls.js

```diff
@@ -1,19 +1,20 @@
-import React, {useEffect} from 'react';
-import {View} from 'react-native';
-import {TouchableOpacity} from 'react-native-gesture-handler';
+import React from 'react';
 import Modal from 'react-native-modal';
 import Colors from '../../../constants/Theme';
+import {ControlBar, ResponseBar} from '../index';
 import styles from './styles';
 
 const VideoControls = (props) => {
-  const {showControls} = props;
+  const {showControls, showResponse} = props;
 
   return (
-    <Modal isVisible={showControls} style={{margin: 0, flex: 1}}>
-      <View style={{flex: 1}}>
-        <View style={{flex: 0.9}} />
-        <View style={{flex: 0.1, backgroundColor: Colors.blue}}></View>
-      </View>
+    <Modal
+      animationIn="fadeIn"
+      animationOut="fadeOut"
+      isVisible={showControls}
+      style={styles.modalContainer}
+      backdropColor={Colors.backdrop}>
+      {showResponse ? <ResponseBar {...props} /> : <ControlBar {...props} />}
```

Comment:
Do you think showControls and showResponse could have been handled with one flag?

Reply:
I had thought for these variables as well.
Problem is that we have three controls - no controls, control options & response options.
To be aligned with all these three options we must have to use 2 variables.
Another options is that -
we take a variable that contains a string, whose value can be - 'noOptions', 'options', 'response'.
But then, I preferred to use boolean variable instead of a string variable, because string variables takes time to compare as compared with boolean variables.